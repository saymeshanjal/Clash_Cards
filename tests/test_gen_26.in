7
4 16 2
12 244 4
16 169 5
16 127 5
10 338 1
12 221 3
12 154 1
