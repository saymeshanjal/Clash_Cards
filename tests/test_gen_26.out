1 2 3
208 441 275
4248 8622 0
4248 8622 0
126 126 0
430 494 0
462 462 0
